NEWS.md

```diff
@@ -6,7 +6,7 @@
 
 - The new option `options(blogdown.knit.serve_site = FALSE)` can be used to prevent **blogdown** from starting the web server automatically when the Knit button is clicked in RStudio and the site has not been served yet (thanks, @Athanasiamo, #572). By default, the web server will be started (if not already started) so the page being knitted can be previewed.
 
-- Added a new global option `blogdown.markdown.format` to allow users to customize Pandoc's Markdown output format. When using the file extension `.Rmarkdown` or `options(blogdown.method = 'markdown')`, an R Markdown file is first compiled to Markdown. This Markdown file needs to go through another conversion when it contains Markdown features that are only available to Pandoc but not other Markdown renderers such as Hugo/Goldmark, such as citations or fenced Divs. The new global option controls the Pandoc output format. By default, its value is `c('gfm', '+footnotes', '+tex_math_dollars')` when the Markdown document contains bibliography or fenced Divs, otherwise it is `NULL`. If you want the conversion to be always performed, you may set this global option to a value that is not `NULL`, e.g., `options(blogdown.markdown.format = c('gfm', '+footnotes', '+tex_math_dollars', '+smart'))`.
+- Added a new global option `blogdown.markdown.format` to allow users to customize Pandoc's Markdown output format. When using the file extension `.Rmarkdown` or `options(blogdown.method = 'markdown')`, an R Markdown file is first compiled to Markdown. This Markdown file needs to go through another conversion when it contains Markdown features that are only available to Pandoc but not other Markdown renderers such as Hugo/Goldmark, such as citations or fenced Divs. The new global option controls the Pandoc output format. By default, its value is `c('gfm', '+footnotes', '+tex_math_dollars')` for Pandoc 2.10.1 and later when the Markdown document contains bibliography or fenced Divs, otherwise it is `NULL`. With earlier version of Pandoc, it will be `c('gfm')` only. If you want the conversion to be always performed, you may set this global option to a value that is not `NULL`, e.g., `options(blogdown.markdown.format = c('gfm', '+footnotes', '+tex_math_dollars', '+smart'))`.
```

Comment:
```suggestion
- Added a new global option `blogdown.markdown.format` to allow users to customize Pandoc's Markdown output format. When using the file extension `.Rmarkdown` or `options(blogdown.method = 'markdown')`, an R Markdown file is first compiled to Markdown. This Markdown file needs to go through another conversion when it contains Markdown features that are only available to Pandoc but not other Markdown renderers such as Hugo/Goldmark, such as citations or fenced Divs. The new global option controls the Pandoc output format. By default, its value is `c('gfm', '+footnotes', '+tex_math_dollars')` for Pandoc 2.10.1 and later when the Markdown document contains bibliography or fenced Divs, otherwise it is `NULL`. With earlier versions of Pandoc, it will be `c('gfm')` only. If you want the conversion to be always performed, you may set this global option to a value that is not `NULL`, e.g., `options(blogdown.markdown.format = c('gfm', '+footnotes', '+tex_math_dollars', '+smart'))`.
```